test/fixtures/flow/type-exports/interface/actual.js

```diff
@@ -1,2 +1,2 @@
 export interface foo { p: number };
-export interface foo<T> { p: T };
+export interface bar<T> { p: T };
```

Comment:
I'm pretty new to Flow - let me know if this isn't the correct fix for this test